core/include/array/array_schema.h

```diff
@@ -685,7 +688,7 @@ class ArraySchema {
    *    - TILEDB_RLE
    *    - TILEDB_BZIP2
    */
-  std::vector<int> compression_;
+  std::vector<Compressor> compression_;
```

Comment:
A global (but perhaps minor) comment: Sometimes we use `compression` and other times we use `compressor`. It would be great if we were consistent.